README.md

```diff
@@ -119,22 +119,25 @@ Note: The axe tests require `prefers-reduced-motion` emulation to be turned on i
 
 This repo uses the [@brightspace-ui/visual-diff utility](https://github.com/BrightspaceUI/visual-diff/) to compare current snapshots against a set of golden snapshots stored in source control.
 
-The golden snapshots in source control must be updated by Github Actions.  If your PR's code changes result in visual differences, a PR with the new goldens will be automatically opened for you against your branch.
+The golden snapshots in source control must be updated by the [visual-diff GitHub Action](https://github.com/BrightspaceUI/actions/tree/main/visual-diff).  If a pull request results in visual differences, a draft pull request with the new goldens will automatically be opened against its branch.
 
-If you'd like to run the tests locally to help troubleshoot or develop new tests, you can use these commands:
+To run the tests locally to help troubleshoot or develop new tests, first install these dependencies:
 
 ```shell
-# Install dependencies locally
-npm install mocha puppeteer @brightspace-ui/visual-diff --no-save
+npm install @brightspace-ui/visual-diff@X mocha@Y puppeteer@Z  --no-save
+```
 
-# run visual-diff tests
-npx mocha './**/*.visual-diff.js' -t 10000
+Replace `X`, `Y` and `Z` with [the current versions](https://github.com/BrightspaceUI/actions/tree/main/visual-diff#current-dependency-versions) the action is using.
 
-# subset of visual-diff tests:
-npx mocha './**/*.visual-diff.js' -t 10000 -g some-pattern
+Then run the tests:
 
+```shell
+# run visual-diff tests
+npx mocha './test/**/*.visual-diff.js' -t 10000
+# subset of visual-diff tests:
+npx mocha './test/**/*.visual-diff.js' -t 10000 -g some-pattern
 # update visual-diff goldens
-npx mocha './**/*.visual-diff.js' -t 10000 --golden
+npx mocha './test/**/*.visual-diff.js' -t 10000 --golden
```

Comment:
I think we want to keep the `./test` out of our README, since we organize tests in each component repo

Reply:
Oops yep thanks!